cpp/ql/src/Likely Bugs/Format/NonConstantFormat.ql

```diff
@@ -146,13 +131,11 @@ module NonConstFlowConfig implements DataFlow::ConfigSig {
 
 module NonConstFlow = TaintTracking::Global<NonConstFlowConfig>;
 
-from FormattingFunctionCall call, Expr formatString
+from FormattingFunctionCall call, Expr formatString, DataFlow::Node sink
 where
   call.getArgument(call.getFormatParameterIndex()) = formatString and
-  exists(DataFlow::Node sink |
-    NonConstFlow::flowTo(sink) and
-    isSinkImpl(sink, formatString)
-  )
-select formatString,
+  NonConstFlow::flowTo(sink) and
+  isSinkImpl(sink, formatString)
+select formatString, //sink.getNode(), src, sink,
```

Comment:
Comment needs cleaning up (until we're ready to convert this into a `path-problem` query?)

Reply:
Fixed